Packs/FeedAzureADConnectHealth/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [Unreleased]
```

Comment:
not needed anymore, can remove.
https://xsoar.pan.dev/docs/integrations/changelog

Reply:
Done